doc/source/whatsnew/v0.18.0.txt

```diff
@@ -161,6 +161,46 @@ In addition, ``.round()`` will be available thru the ``.dt`` accessor of ``Serie
 
 .. _whatsnew_0180.api_breaking:
 
+
+Change in to_string(index=False) method of DataFrameFormatter and SeriesFormatter class
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+- the "leading whitespaces" have been removed from the output of "to_string(index=False)" method (:issue: '11833')
+
+Previous Behaviour for DataFrame Objects
+
+.. ipython:: python
+
+   df=pd.DataFrame({'a':range(5),'b':range(5)})
+   df.to_string(index=False)
+   u' a  b\n 0  0\n 1  1\n 2  2\n 3  3\n 4  4'
```

Comment:
you need to mark this ``code-block:: python``, then paste in actual ipython output